# Status of traffic lights in system. Namely, node_ids
# of the traffic lights and whether they are on/off.

int32[] node_ids
bool[] on